### ISSUE TITLE
Modify Loss function 

### ISSUE THREAD

**Issue by meetshah1995** [meetshah1995/pytorch-semseg#8]
Categorical Crossentropy 2D implemented currently assumes 0th index as the default background (ignore label). Make the ignore label as configurable argument and an attribute of the dataset.
